packages/integration-tests/src/tests/api/sso-connectors.test.ts

```diff
@@ -10,3 +12,80 @@ describe('sso-connector', () => {
     expect(response.standardConnectors.length).toBeGreaterThan(0);
   });
 });
+
+describe('post sso-connectos', () => {
+  it('should throw error when providerName is not provided', async () => {
+    await expect(
+      createSsoConnector({
+        connectorName: 'test',
+      })
+    ).rejects.toThrow(HTTPError);
+  });
+
+  it('should throw error when connectorName is not provided', async () => {
+    await expect(
+      createSsoConnector({
+        providerName: 'OIDC',
+      })
+    ).rejects.toThrow(HTTPError);
+  });
+
+  it('should throw error when providerName is not supported', async () => {
+    await expect(
+      createSsoConnector({
+        providerName: 'SAML',
```

Comment:
nit: better use some other name like `OpenIDConnect` since we are supporting SAML as providerName and this case will be easily broken.

Reply:
good point

Reply:
Updated